llama_stack/providers/registry/vector_io.py

```diff
@@ -45,7 +47,7 @@ def available_providers() -> List[ProviderSpec]:
         InlineProviderSpec(
             api=Api.vector_io,
             provider_type="inline::sqlite_vec",
-            pip_packages=["sqlite-vec"],
+            pip_packages=["sqlite-vec --no-binary=sqlite-vec"],
```

Comment:
is the `--no-binary=sqlite-vec` on purpose?

Reply:
oh weird, straggler change; I should have removed it.